net/xia/ppal_lpm/main.c

```diff
@@ -136,6 +137,7 @@ static int newroute_flush_anchor_unlock(struct fib_xid_table *xtbl,
 		}
 
 		dup_mrd->gw = fxid_mrd(pred_fxid)->gw;
+		lpm_rt_iops->fxid_copy(&dup_mrd->common, pred_fxid);
```

Comment:
If you've inverted the parameters of `fxid_copy()` above, you fixed another bug here.